### ISSUE TITLE
Capture sambamba output and pipe to logfile

### ISSUE THREAD

**Issue by robinandeer** [robinandeer/chanjo#157]
If one runs sambamba through chanjo it would be awesome if the STDOUT/ERR would redirect to the same logfiles, don't you think?

**Comment by moonso** [comment]
Sure I'll fix this

**Comment by moonso** [comment]
If I understand correct this is already done. If chanjo is run with a log file the sambamba output will be written here.
See:
```python
def run_sambamba(bam_file, region_file, outfile=None, cov_treshold=()):
    """Run sambamba from chanjo."""
    logger = logging.getLogger(__name__)
    logger = logging.getLogger("chanjo.sambamba")
    log_stream = get_log_stream(logger)
    sambamba_call = [
        "sambamba",
        "depth",
        "region",
        "--regions",
        region_file,
        bam_file
    ]
```

The logstream-handler should take care of this...

**Comment by robinandeer** [comment]
Indeed, it does seem like it should already handle this.. Can you think of a reason why the "chanjo.sambamba" logger wouldn't be bound to a file handler? That's the only reason I see why it doesn't work :P

**Comment by moonso** [comment]
I think that the log handler sets up a ```FileHandler``` when init_log is called with a file.
Otherwise the sambamba logging will be printed to stderr as expected
